### DISCUSSION TITLE
Welcome to Garnet Discussions!

### DISCUSSION THREAD

**Discussion by badrishc** [microsoft/garnet#30] [Announcements]
## 👋 Welcome!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 15012021** [comment]
💪

**Comment by Sean-Yang-Carpediem** [comment]
Does garnet have 16 repositories like redis?

**Reply by wessisaac** [reply]
At present Garnet has one repository (database), as per their replies, they are working for multi database on standalone version.
